front/app/components/FormBuilder/components/FormBuilderToolbox/ToolboxItem.tsx

```diff
@@ -52,40 +82,66 @@ const ToolboxItem = ({
   disabled,
   ...rest
 }: Props) => {
+  const { formatMessage } = useIntl();
+
   if (fieldsToExclude && inputType && fieldsToExclude.includes(inputType)) {
     return null;
   }
 
   return (
-    <StyledBox
-      display="flex"
-      px="18px"
-      py="18px"
-      onClick={onClick}
-      width="100%"
-      m="0px"
-      alignItems="center"
-      // remove the role attribute when we add drag and drop functionality
-      role="button"
-      data-cy={rest['data-cy']}
-      disabled={!!disabled}
+    <Tippy
+      interactive={true}
+      placement={'left'}
+      disabled={!disabled}
+      theme={'dark'}
+      content={
+        <ContentWrapper id="tooltip-content" tippytheme={'light'}>
+          {formatMessage(messages.disabledFieldTooltip)}
```

Comment:
I think we'll have a need for tooltips in other participation methods going forwards, but maybe the message should be from the config instead? :thinking: